src/containers/AlbumsView.jsx

```diff
@@ -34,7 +41,10 @@ export class AlbumsView extends Component {
       !this.props.children
         ? <div>
           <Topbar viewName='albums' />
-          <AlbumsList {...this.props} {...this.state} />
+          <AlbumsList
+            {...this.props}
+            handleErrorToParent={this.handleErrorFromChildren}
```

Comment:
As mentioned above, I think this attribute should be named `onError` (cc @goldoraf).